### DISCUSSION TITLE
Feature Request: Add functionality to dashboard navigation panel enabling users to translate variables between dashboards

### DISCUSSION THREAD

**Discussion by robhamnett** [grafana/grafana#63320] [Dashboards, Panels & Visualizations]
When utilizing dashboard navigation panels and dropdown variables, users are unable to translate variables between dashboards when navigating between dashboards. An option to append variables and time series info to selected dashboards when navigating would greatly improve user navigation.

![image](https://user-images.githubusercontent.com/65688161/218552773-a1f42d87-2057-44c5-aa30-e48c9f3a7381.png)

In this use case, the site variable represents locations across our network. Selecting another dashboard from the dashboard navigation panel uses the default value for a given variable and not the currently selected value (even tho the site variable is present across all dashboards). This can be easily accomplished using data links and appending to the URL however using the dashboard navigation panel, translation of variables is currently not supported.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are https://github.com/grafana/grafana/discussions/73424. I believe this is being addressed here https://github.com/grafana/grafana/issues/1032

If I am mistaken, feel free to make an [issue](https://github.com/grafana/grafana/issues) if you cannot find a solution and would like to request the feature.
